elegy/nets/resnet.py

```diff
@@ -2,49 +2,80 @@
 
 import jax, jax.numpy as jnp
 from elegy import module, nn
+import typing as tp
+
+
+__all__ = ["ResNet18", "ResNet34", "ResNet50", "ResNet101", "ResNet152", "ResNet200"]
```

Comment:
Should we add the base `ResNet` as well?

Reply:
Done